vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -12,7 +12,7 @@ $ErrorActionPreference = "Stop"
 
 filter Timestamp {"$(Get-Date -Format o): $_"}
 
-$global:containerdPackageUrl = "https://marosset.blob.core.windows.net/pub/containerd/containerd-0.0.87-public.zip"
+$global:containerdPackageUrl = "https://k8swin.blob.core.windows.net/k8s-windows/containerd/containerplat-aks-test-0.0.8.zip"
 
```

Comment:
Is this a package built by yourself from https://github.com/jterry75/cri.git?

Reply:
Could we also copy ` scripts/build-windows-containerd.sh` to agentbaker later?

Reply:
> Is this a package built by yourself from https://github.com/jterry75/cri.git?

No, it's from [kubernetes-hybrid.kubenet-containerd example of James' hyper-v PR](https://github.com/Azure/aks-engine/pull/3688/files#diff-49031e7602941ee914049a58812b60efR10)
I prefer not cache the test containerD package in the vhd, but use cse to install the containerD of the correct version. 

> Could we also copy scripts/build-windows-containerd.sh to agentbaker later?

I think so, but I'd prefer have this in another work, together with a pipeline and doc to build self-owned containerd.

Reply:
For test purpose, it is ok but later we should cache it for the final vhd